docs/docs/modeling.md

```diff
@@ -106,6 +106,41 @@ classDiagram
 
 ## ShareHouse - シェアハウス
 
+The Landlord can register the ShareHouse owned by the Landlord.
+
+### ShareHouse ID (Identifier)
+
+The ShareHouse ID is the UUID.
+
+### ShareHouse Name
+
+The name of the ShareHouse can be set and changed.
+The ShareHouse has a default name when created without specifying one.
+
+Constraints：The ShareHouse Name must be
+
+- at least 1 character and no more than 15 characters long.
+
+### rotationCycle
+
+The rotationCycle must be set to Weekly or Fortnightly.
+
+Constraints：
+
+- Tenants cannot be assigned to a Category until the rotationCycle is set.
+
+### assignmentSheet
+
+Tenant's task assignment table.
```

Comment:
IMO, to align with the wording used in other sentences, I believe it would be beneficial to include the subject (assignmentSheet). What are your thoughts?

The assignmentSheet is the tenant's task assignment table.

Reply:
Thank you for pointing that out. I agree with you so I will revise the sentence.